### DISCUSSION TITLE
Loss of communication between Manager and Search nodes after so-features-enable

### DISCUSSION THREAD

**Discussion by Hemro** [Security-Onion-Solutions/securityonion#2709] [Unsupported Versions]
I have used the so-feature-enable command because there is a requirement for the Canvas feature in kibana. Unfortunately after running this it changes communication to http which is fine for my environment. It now seems that it is still trying to send data via https and I cannot find anywhere to swap it over to http. Any assistant would be grateful appreciated.

[2021-01-20T11:49:39,424][INFO ][logstash.outputs.elasticsearch] Elasticsearch pool URLs updated {:changes=>{:removed=>[], :added=>[https://192.168.80.151:9200/]}}
[2021-01-20T11:49:39,737][ERROR][logstash.javapipeline    ] Pipeline error {:pipeline_id=>"search", :exception=>#<Manticore::UnknownException: Unsupported or unrecognized SSL message>,

**Comment by weslambert** [comment]
What version of Security Onion are you running?

**Comment by Hemro** [comment]
2.3.21

**Reply by weslambert** [reply]
Is this a standalone or distributed architecture?

**Reply by Hemro** [reply]
distributed architecture

**Reply by weslambert** [reply]
Thanks, we are looking into this now.

**Reply by cm-ops** [reply]
@Hemro 

Here is what I did to get ES/LS working. 

Verify your output files in `/opt/so/conf/logstash/pipelines/search` on your search node.

Edit the output files on the manager in `/opt/so/saltstack/default/salt/logstash/pipelines/config/so` changing `ssl => true` to `ssl => false`.

Run a highstate and check the ES/LS logs on the search node to verify the services are up and running.
